forest/daemon/Cargo.toml

```diff
@@ -82,10 +82,8 @@ tokio                            = { workspace = true, features = ["sync"] }
 toml.workspace                   = true
 
 [dev-dependencies]
-assert_cmd.workspace        = true
```

Comment:
@lemmih Looks like the `align_entries` rule always creates diff noices for such cases, do we really need that rule?

Reply:
Hmm, I guess aligning the dependencies is less relevant when they're all inherited from the workspace. I still prefer the noise, though.